OptionSelector/CategoryTableViewController.m

```diff
@@ -0,0 +1,118 @@
+//
+//  CategoryTableViewController.m
+//  OptionSelector
+//
+//  Created by Henna on 8/10/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CategoryTableViewController.h"
+#import "CQCategory.h"
+#import "OptionsTableViewController.h"
+#import "CategoryModel.h"
+
+
+@interface CategoryTableViewController ()
+@property (nonatomic) CategoryModel * data;
+
+@end
+
+@implementation CategoryTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.title = @"Henna's Categories";
+    self.data = [CategoryModel sharedInstance];
+    [self.data initializeModel];
```

Comment:
Again, because `self.data` is a reference to the `CategoryModel` singleton, you probably don't want to do this here. You should just call `initializeModel once when the singleton instance is allocated/initialized, rather than every time this view controller's `viewDidLoad` method is called.